app/src/main/java/nl/tudelft/bbw/blockchain/Block.java

```diff
@@ -280,7 +280,7 @@ public boolean verifyBlock(Object o) {
     @Override
     public String toString() {
         return "Block#" + getSequenceNumber() + "{owner=" + owner.getName() + ",contact="
-                + contact.getName() + "}";
+                + contact.getName() + (isRevoked()?",REVOKE":"") + "}";
```

Comment:
I have ran it with gradlew build connectedCheck check lint test yet I see no problem here. You can try.